bin/improver-spot-extract

```diff
@@ -171,8 +171,10 @@ def main():
                 if not args.suppress_warnings:
                     warnings.warn(msg)
         else:
+            print(args.extract_percentile)
```

Comment:
You probably didn't mean to leave these print statements in

Reply:
Agreed.